src/components/header/index.tsx

```diff
@@ -1,42 +1,46 @@
+import useFlowStore, { flowSelector } from "@pb/store/flow-builder.store";
 import Link from "next/link";
 import { useEffect, useState } from "react";
 import { themeChange } from "theme-change";
+import { shallow } from "zustand/shallow";
 
 const themeOptions = [
   { value: "", label: "Default" },
-  { value: "light", label: "Light" },
-  { value: "dark", label: "Dark" },
-  { value: "cupcake", label: "Cupcake" },
+  { value: "acid", label: "Acid" },
+  { value: "aqua", label: "Aqua" },
+  { value: "autumn", label: "Autumn" },
+  { value: "black", label: "Black" },
   { value: "bumblebee", label: "Bumblebee" },
-  { value: "emerald", label: "Emerald" },
+  { value: "business", label: "Business" },
+  { value: "cmyk", label: "Cmyk" },
+  { value: "coffee", label: "Coffee" },
   { value: "corporate", label: "Corporate" },
-  { value: "synthwave", label: "Synthwave" },
-  { value: "retro", label: "Retro" },
+  { value: "cupcake", label: "Cupcake" },
   { value: "cyberpunk", label: "Cyberpunk" },
-  { value: "valentine", label: "Valentine" },
-  { value: "halloween", label: "Halloween" },
-  { value: "garden", label: "Garden" },
+  { value: "dark", label: "Dark" },
+  { value: "dracula", label: "Dracula" },
+  { value: "emerald", label: "Emerald" },
+  { value: "fantasy", label: "Fantasy" },
   { value: "forest", label: "Forest" },
-  { value: "aqua", label: "Aqua" },
+  { value: "garden", label: "Garden" },
+  { value: "halloween", label: "Halloween" },
+  { value: "lemonade", label: "Lemonade" },
+  { value: "light", label: "Light" },
   { value: "lofi", label: "Lofi" },
-  { value: "pastel", label: "Pastel" },
-  { value: "fantasy", label: "Fantasy" },
-  { value: "wireframe", label: "Wireframe" },
-  { value: "black", label: "Black" },
   { value: "luxury", label: "Luxury" },
-  { value: "dracula", label: "Dracula" },
-  { value: "cmyk", label: "Cmyk" },
-  { value: "autumn", label: "Autumn" },
-  { value: "business", label: "Business" },
-  { value: "acid", label: "Acid" },
-  { value: "lemonade", label: "Lemonade" },
   { value: "night", label: "Night" },
-  { value: "coffee", label: "Coffee" },
+  { value: "pastel", label: "Pastel" },
+  { value: "retro", label: "Retro" },
+  { value: "synthwave", label: "Synthwave" },
+  { value: "valentine", label: "Valentine" },
   { value: "winter", label: "Winter" },
+  { value: "wireframe", label: "Wireframe" },
 ];
 
 const Header: React.FC = () => {
   const [theme, setTheme] = useState("");
+  const { updateFlowMode, flowMode } = useFlowStore(flowSelector, shallow);
+  const showChat = flowMode === 1;
```

Comment:
updated

Reply:
> having a enum as a number will help controlling this from the DB, but since we use Prisma we can change it to be a string enum

I agree with you. It is important to choose meaningful names that accurately reflect the intended functionality. However, regarding my previous comment, I suggested changing the constant name **showChat** to either **isTestModeEnabled** or **isTestModeActivated**.